app/src/main/java/cm/aptoide/pt/AptoideApplicationAnalytics.java

```diff
@@ -30,4 +30,12 @@ public void setVersionCodeDimension(String versionCode) {
     AppEventsLogger.updateUserProperties(bundle, response -> Logger.getInstance()
         .d("Facebook Analytics: ", response.toString()));
   }
+
+  public void setIronSourceAbTestGroup(boolean isControlGroup) {
+    Bundle bundle = new Bundle();
+    bundle.putString("ab_test_ironsource",
+        isControlGroup ? "a_without_ironsource" : "b_with_ironsource");
+    AppEventsLogger.updateUserProperties(bundle, response -> Logger.getInstance()
```

Comment:
Won't this only be updated if the user goes to the appview? If yes won't this bring problems to the analysis?